src/C_API.jl

```diff
@@ -557,6 +561,9 @@ function solve_mcp(
     variable_names::Vector{String} = String[],
     constraint_names::Vector{String} = String[],
     silent::Bool = false,
+    generate_output = 0,
+    use_start = true,
+    use_basics = false,
```

Comment:
These need type parameters, and to be added to the docstring.

Reply:
Done